code/modules/clothing/spacesuits/_spacesuits.dm

```diff
@@ -85,6 +85,8 @@
 /obj/item/clothing/suit/space/equipped(mob/living/user, slot)
 	. = ..()
 	if(slot & ITEM_SLOT_OCLOTHING) // Check that the slot is valid
+		if(isplasmaman(user))
+			slowdown = 1
```

Comment:
Undo this.
It's fine for plasmamen helmets to not have slowdown, but this isn't necessary.